src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -1148,6 +1148,99 @@ RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** fog ********************/
+
+RppStatus rppt_fog_host(RppPtr_t srcPtr,
+                        RpptDescPtr srcDescPtr,
+                        RppPtr_t dstPtr,
+                        RpptDescPtr dstDescPtr,
+                        Rpp32f *gammaTensor,
+                        RpptROIPtr roiTensorPtrSrc,
+                        RpptRoiType roiType,
+                        rppHandle_t rppHandle)
+{
+    RppLayoutParams layoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+
+    RpptDesc fogMaskSrcDesc; 
+    RpptDescPtr fogMaskSrcDescPtr = &fogMaskSrcDesc; // Pointer to original fog mask source descriptor
+    set_fog_mask_descriptor(fogMaskSrcDescPtr, 2, FOG_MAX_HEIGHT, FOG_MAX_WIDTH, 1);
```

Comment:
Move the comments to the top of the code which it corresponds to
```
// Pointer to original fog mask source descriptor
RpptDescPtr fogMaskSrcDescPtr = &fogMaskSrcDesc; 
```

Inside the kernel file, we can probably add the comments sidewise
Applicable to all such instances in the code section